openff/toolkit/utils/openeye_wrapper.py

```diff
@@ -1203,6 +1202,19 @@ def from_openeye(
 
         oemol = oechem.OEMol(oemol)
 
+        components = oechem.OEDetermineComponents(oemol)
+        if components[0] > 1:
+            warnings.warn("OpenEye OEMol passed to from_openeye consists of more than one molecule, consider running "
```

Comment:
Perhaps a custom warning to make _this specific one_ easier for users to filter out? Thinking about cases like SPICE in which being warned for each ~offending molecule would be spammy

Reply:
Good call, added!